### DISCUSSION TITLE
AQV12MSAN IS THERE ANY SOLUTION?

### DISCUSSION THREAD

**Discussion by hanskiepl** [omerfaruk-aran/esphome_samsung_hvac_bus#139] [Questions]
Hi, I've got oldest model as described in the topic. Is that possible to control this unit via HomeAssistan?

**Comment by lanwin** [comment]
@hanskiepl currently there is no one how can tell you if this works cause no one reported this unit to work. You have to try it.

Please look at the readme and check if the device has the ports described and if yes, then you need to buy the hardware and check if it works.

**Reply by hanskiepl** [reply]
![20240701_195851.jpg](https://github.com/lanwin/esphome_samsung_ac/assets/52385597/c2587285-c83f-4622-bb0d-4fc0166515e9)

I don't think there are connectors described in readme.

**Reply by lanwin** [reply]
If there is no F1/F2 then it dont work. We already have one case where there are only power wires and nothing else "AR12BSEAMWKX (no communication wires)"

**Comment by Goaheadz** [comment]
I have old AQV18PSBN and it works fine.
it shouldnt be diferent from AQV12**MSAN**

Pull this cover off:
![image](https://github.com/user-attachments/assets/0d1a5e1d-fa03-4fa0-bd53-9baacb02abaa)
You should see F1 and F2
![image](https://github.com/user-attachments/assets/f5843028-53a4-4727-840b-8eb48aa5cf14)
